numpy/linalg/linalg.py

```diff
@@ -1909,6 +1916,19 @@ def lstsq(a, b, rcond=-1):
     if m != b.shape[0]:
         raise LinAlgError('Incompatible dimensions')
     t, result_t = _commonType(a, b)
+    # Determine default rcond value
+    if rcond == "warn":
+        warnings.warn("`rcond` parameter will change to the default of "
```

Comment:
A comment with the date and version of this change will be helpful for future maintenance. Something like
```
# 2017-08-19, 1.14.0
```